tasks/packages.yml

```diff
@@ -8,19 +8,23 @@
     group: root
     mode: 0755
   with_items:
+    - src: R-list-installed-packages.j2
+      dest: R-list-installed-packages
     - src: R-install-package.j2
       dest: R-install-package
     - src: R-remove-package.j2
       dest: R-remove-package
     - src: R-update-package.j2
       dest: R-update-package
-    - src: R-update-packages.j2
-      dest: R-update-packages
```

Comment:
Why is this removed

Reply:
It isn't used anywhere in the role, but ok, I restored it.